client/packages_test.go

```diff
@@ -339,11 +352,20 @@ func (cs *clientSuite) TestClientSnap(c *check.C) {
 		Links: map[string][]string{
 			"website": {"http://example.com/funky"},
 		},
-		Website:  "http://example.com/funky",
-		StoreURL: "https://snapcraft.io/chatroom",
+		Website:                   "http://example.com/funky",
+		StoreURL:                  "https://snapcraft.io/chatroom",
+		RefreshInhibitProceedTime: expectedRefreshInhibitProceedTime,
 	})
 }
 
+func (cs *clientSuite) TestClientSnap(c *check.C) {
+	cs.testClientSnap(c, false)
```

Comment:
this is slightly clearer as:

const refreshInhibited = false
cs.testClientSnap(c, refreshInhibited )